crystalball.gemspec

```diff
@@ -41,11 +41,12 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'parser'
   spec.add_development_dependency 'pry'
   spec.add_development_dependency 'pry-byebug'
+  spec.add_development_dependency 'psych', '~> 4.0.0'
   spec.add_development_dependency "rake", "~> 10.0"
   spec.add_development_dependency "rspec", "~> 3.0"
   spec.add_development_dependency 'rubocop', ">= 0.56"
   spec.add_development_dependency 'rubocop-rspec'
   spec.add_development_dependency 'simplecov'
-  spec.add_development_dependency 'sqlite3', "~> 1.3.13"
+  spec.add_development_dependency 'sqlite3', "~> 1.4.2"
```

Comment:
An earlier version of sqlite was not building on my Mac, this upgrade was to unblock me from running the setup and the tests.